backend/src/test/java/heartbeat/service/report/KanbanFixture.java

```diff
@@ -0,0 +1,41 @@
+package heartbeat.service.report;
+
+import heartbeat.controller.board.dto.request.RequestJiraBoardColumnSetting;
+import heartbeat.controller.board.dto.request.StoryPointsAndCycleTimeRequest;
+import heartbeat.controller.board.dto.response.TargetField;
+import heartbeat.controller.report.dto.request.JiraBoardSetting;
+
+import java.util.List;
+
+public class KanbanFixture {
+
+	public static JiraBoardSetting MOCK_JIRA_BOARD_SETTING() {
+		return JiraBoardSetting.builder()
+			.users(List.of("user1"))
+			.token("token")
+			.type("jira")
+			.site("site")
+			.projectKey("ADM")
+			.boardId("2")
```

Comment:
duplicated code with line 29 ~ line 33

Reply:
different types of object definitions, do not know how to repeat